<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1880</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="details_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>730</y>
      <width>1841</width>
      <height>171</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Directory</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>1461</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="overview_tab">
     <attribute name="title">
      <string>Overview</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1441</width>
        <height>651</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="overview_grid">
       <item row="3" column="1">
        <widget class="QTextBrowser" name="ov_text">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Text field&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="ov_plot2_layout">
         <item row="1" column="1">
          <widget class="QGraphicsView" name="plot_window_3"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="Xaxis_selector_3">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Time (s)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="Yaxis_selector_3">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <item>
            <property name="text">
             <string>Magnitude (arb)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="plot_label_3">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Plot Label</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QComboBox" name="plot_selector_3">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <item>
            <property name="text">
             <string>Plot1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Plot2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="param_table">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="cell">
           <item>
            <widget class="QComboBox" name="parameter_box">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="value">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="cell_5">
           <item>
            <widget class="QComboBox" name="parameter_box_5">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="value_5">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="cell_3">
           <item>
            <widget class="QComboBox" name="parameter_box_3">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="value_3">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="cell_4">
           <item>
            <widget class="QComboBox" name="parameter_box_4">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="value_4">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QVBoxLayout" name="cell_6">
           <item>
            <widget class="QComboBox" name="parameter_box_6">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="value_6">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="cell_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QComboBox" name="parameter_box_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="value_2">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QVBoxLayout" name="cell_7">
           <item>
            <widget class="QComboBox" name="parameter_box_7">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="value_7">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <layout class="QVBoxLayout" name="cell_8">
           <item>
            <widget class="QComboBox" name="parameter_box_8">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="value_8">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="ov_plot1_layout">
         <item row="1" column="1">
          <widget class="QGraphicsView" name="plot_window_2"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="Xaxis_selector_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Time (s)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="Yaxis_selector_2">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <item>
            <property name="text">
             <string>Magnitude (arb)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="plot_label_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Plot Label</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QComboBox" name="plot_selector_2">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <item>
            <property name="text">
             <string>Plot1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Plot2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Table</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>941</width>
        <height>601</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="load_dir_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load from Directory</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="selection_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>381</width>
      <height>601</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <column>
     <property name="text">
      <string>Parent Folder</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Calc Folder</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Files Available</string>
     </property>
    </column>
   </widget>
   <widget class="QListWidget" name="file_ext_boxes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;List of file extensions to filter to&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="accessibleName">
     <string>List of file types</string>
    </property>
    <property name="flow">
     <enum>QListView::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>.in</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.out</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.txt</string>
     </property>
     <property name="checkState">
      <enum>Checked</enum>
     </property>
    </item>
    <item>
     <property name="text">
      <string>_</string>
     </property>
     <property name="checkState">
      <enum>Unchecked</enum>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1880</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
